packages/zostso/src/IssueTso.ts

```diff
@@ -34,50 +130,35 @@ export class IssueTso {
      * @memberof IssueTso
      */
-    public static async issueTsoCommand(session: AbstractSession, accountNumber: string, command: string, startParams?: IStartTsoParms) {
-
-        TsoValidator.validateSession(session);
-        TsoValidator.validateNotEmptyString(accountNumber, noAccountNumber.message);
-        TsoValidator.validateNotEmptyString(command, noCommandInput.message);
-
-        const response: IIssueResponse = {
-            success: false,
-            startResponse: null,
-            startReady: false,
-            zosmfResponse: null,
-            commandResponse: null,
-            stopResponse: null
-        };
-        response.startResponse = await StartTso.start(session, accountNumber, startParams || {});
-
-        if (!response.startResponse.success) {
-            throw new ImperativeError({
-                msg: `TSO address space failed to start.`,
-                additionalDetails: response.startResponse.failureResponse?.message
-            });
-        }
-
-        const sendResponse = await SendTso.sendDataToTSOCollect(session, response.startResponse.servletKey, command);
-        response.success = sendResponse.success;
-        response.zosmfResponse = sendResponse.zosmfResponse;
-        response.commandResponse = sendResponse.commandResponse;
-        response.stopResponse = await StopTso.stop(session, response.startResponse.servletKey);
-        return response;
+    public static async issueTsoCommand(
+        session: AbstractSession,
+        accountNumber: string,
+        command: string,
+        startParams?: IStartTsoParms
+    ): Promise<IIssueResponse> {
+        return await IssueTso.issueTsoCmd(session, command, {
+            suppressStartupMessages: false,
+            addressSpaceOptions: { ...startParams, account: accountNumber },
+        });
     }
 
     /**
+     * @deprecated use issueTsoCmd instead
      * API method to start a TSO address space with provided parameters, issue a command,
      * collect responses until prompt is reached, and terminate the address space.
      * @param {AbstractSession} session - z/OSMF connection info
      * @param {IIssueTsoParms} commandParms - object with required parameters, @see {IIssueTsoParms}
      * @returns {Promise<IIssueResponse>}
      */
-    public static async issueTsoCommandCommon(session: AbstractSession, commandParms: IIssueTsoParms) {
-
-        TsoValidator.validateSession(session);
-        TsoValidator.validateIssueParams(commandParms);
-        TsoValidator.validateNotEmptyString(commandParms.command, noCommandInput.message);
-        return IssueTso.issueTsoCommand(session, commandParms.accountNumber, commandParms.command, commandParms.startParams);
+    public static async issueTsoCommandCommon(
+        session: AbstractSession,
+        commandParms: IIssueTsoParms
+    ): Promise<IIssueResponse> {
+        return await IssueTso.issueTsoCmd(session, commandParms.command, {
+            addressSpaceOptions: {
```

Comment:
After some more thought, I'm curious if we should keep backwards compatibility in this common method as well.
Meaning, pass the `suppessStartupMessages: false`. 😅 
I do believe this is a less common function as I only saw it in forks of the CLI and in the kotlin-externals. 😅 

Not a hard requirement though.
I kind of want to see if someone complains about this one 😋